src/libraries/Microsoft.PowerFx.Core/IR/Nodes/NumberLiteralNode.cs

```diff
@@ -22,5 +25,22 @@ public override string ToString()
         {
             return $"Number({LiteralValue})";
         }
+
+        public override bool Equals(object obj)
```

Comment:
>Equals [](http://example.com/codeflow?start=29&length=6)

same with IRContext - why do we need this?

Reply:
removed